ofrak_core/ofrak/ofrak_cli.py

```diff
@@ -270,16 +282,69 @@ def handler(ofrak_env: OFRAKEnvironment, args: Namespace):
 
             dependency_info = f"{dep.tool}\n\t{dep.tool_homepage}\n\t[{', '.join(c for c in components_by_dep[dep])}]"
             if not args.no_check:
-                dependency_info = f"[{' ' if not is_installed else '✓'}] " + dependency_info
+                dependency_info = (
+                    f"[{' ' if not is_installed else '✓'}] " + dependency_info
+                )
 
             print(dependency_info)
 
 
+class GUISubCommand(OFRAKSubCommand):
+    def create_parser(self, ofrak_subparsers):
+        gui_parser = ofrak_subparsers.add_parser(
+            "gui",
+            help="Launch the OFRAK GUI server.",
+            description="Launch the OFRAK GUI server.",
+        )
+        gui_parser.add_argument(
+            "-H",
+            "--hostname",
+            action="store",
+            help="Set GUI server host address.",
+            default=None,
+        )
+        gui_parser.add_argument(
+            "-p",
+            "--port",
+            action="store",
```

Comment:
Based on the type of the `_port` variable in `ofrak_server.py`, this should probably have `type=int`. See [these `argparse` docs](https://docs.python.org/3/library/argparse.html#type).

```suggestion
            action="store",
            type=int,
```